src/obb/events/lobby_events.clj

```diff
@@ -0,0 +1,49 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.events.lobby-events
+  "Manages subscribers and processes lobby events"
+  (:require [clojure.core.async :as async]
+            [com.stuartsierra.component :as component]
+            [obb.models.game-info :as game-info]))
+
+(defprotocol LobbyEvents
+  (add-game [this game-info])
+  (remove-game [this game-info])
+  (subscribe [this subscriber-ch]))
+
+(defrecord LobbyEventSystem [master-ch multiplier]
+
+  component/Lifecycle
+
+  (start [component]
+    (let [master-ch (async/chan)
+          multiplier (async/mult master-ch)]
+      (assoc component :master-ch master-ch
+                       :multiplier multiplier)))
+
+  (stop [component]
+    (async/close! master-ch)
+    (-> component
+        (dissoc :master-ch)
+        (dissoc :multiplier)))
+
+  LobbyEvents
+
+  (add-game [this game-info]
+    (async/go
+      (async/>! master-ch
+                {:action :added
```

Comment:
This is really a minor thing. I would stick to using the present form of the verb in this field. If the action is called `add-game`, I'd call the action `add` or really `add-game`. Its just simpler to call them the same thing. Also, I am not sure why the action name is needed, but I'd consider putting it in the `:_metadata` field I suggested in the previous commit.